src/Confluent.Kafka/IProducer.cs

```diff
@@ -339,6 +339,14 @@ public interface IProducer<TKey, TValue> : IClient
         ///     Any produce call outside an on-going transaction, or for a failed
         ///     transaction, will fail.
         /// </remark>
+        /// <exception cref="TransactionRequiresAbortException">
```

Comment:
BeginTransaction will not raise ABORTABLE.
All the transaction APIs have doc strings to say which error types are raised:
https://github.com/edenhill/librdkafka/blob/master/src/rdkafka.h#L6699

Reply:
thanks